src/cargo/ops/cargo_output_metadata.rs

```diff
@@ -74,7 +74,6 @@ impl Encodable for MetadataResolve {
     fn encode<S: Encoder>(&self, s: &mut S) -> Result<(), S::Error> {
         #[derive(RustcEncodable)]
         struct EncodableResolve<'a> {
-            root: &'a PackageId,
```

Comment:
This is a breaking changing regaeding `cargo metadata` json format. We can use arbitrary package as root here, but I'd rather just remove this field. IntelliJ Rust already ignores the `root` field.